### PR TITLE
Feature/convinience message property

### LINKED ISSUE #3075: [FEATURE] Add convenience property `id` to `Message`
## Current situation

Telegram currently offers attribute `message_id` to `Message`. But this is inconsistent with the rest of their API, e.g. `Chat.id`, `User.id`, `InlineQuery.id`, etc. We can offer a convenience property `Message.id` which will make it more consistent and make it less prone to errors, e.g. when someone forgets it's `message_id` and not `id`. 

## Proposed change

Add the property `id` to `Message` which just returns `self.message_id`. A similar example can be found [here](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/telegram/_message.py#L570-L573).

## A PR for this issue should

- Add a docstring along with a `.. versionadded:: 20.0` directive to the function.
- Add an unit test to `test_message.py` that ensures that this works as expected and is the same as `Message.message_id`.

## Additional Info

We have a [contribution guide](https://python-telegram-bot.readthedocs.io/en/v20.0a0/contributing.html)

### PR DESCRIPTION
**Pull request by dglitxh**
### Checklist for PRs

- [x] Added `.. versionadded:: version`, `.. versionchanged:: version` or `.. deprecated:: version` to the docstrings for user facing changes (for methods/class descriptions, arguments and attributes)
- [x] Created new or adapted existing unit tests
- [x] Documented code changes according to the [CSI standard](https://standards.mousepawmedia.com/en/stable/csi.html)
- [x] Added myself alphabetically to `AUTHORS.rst` (optional)
- [ ] Added new classes & modules to the docs and all suitable `__all__` s

closes #3075

### CODE REVIEW

File: telegram/_message.py [hunk 1]
```diff
@@ -572,6 +572,17 @@ def chat_id(self) -> int:
...
         return self.chat.id
 
+    @property
+    def id(self) -> int:  # pylint: disable=invalid-name
+        """
+        .. versionchanged:: 20.0
+          :obj: `int` : Convenience property. This is to reduce errors when users
+          use Message.id instead of Message.message_id which is inconsistent with most
+          of the other properties which follow a similar pattern.
+
```
Comment (harshil21) [inline telegram/_message.py:582]: no need to mention the reason for this addition imo. We can just have something like:

```suggestion
        :obj:`int`: Shortcut for :attr:`message_id`.
        
            .. versionadded:: 20.0
```

**Review by harshil21 — COMMENTED** [review]
Thanks for the quick PR! Just left one comment below-

File: telegram/_message.py (hunk 1 above)
Comment (dglitxh) [reply telegram/_message.py]: That's great! I was having a bit of an issue with that.

**Comment by harshil21** [comment]
pre-commit.ci run

**Pull request merged**